stardis/radiation_field/opacities/opacities_solvers/broadening.py

```diff
@@ -284,32 +288,115 @@ def calc_gamma_quadratic_stark(
     electron_density : float
         Electron density at depth point being considered.
     temperature : float
-        Temperature of depth points being considered.
+        Temperature at depth point being considered.
 
     Returns
     -------
     gamma_quadratic_stark : float
         Broadening parameter for quadratic Stark broadening.
     """
-    c4_prefactor = (ELEMENTARY_CHARGE**2 * BOHR_RADIUS**3) / (
-        36 * PLANCK_CONSTANT * VACUUM_ELECTRIC_PERMITTIVITY * ion_number**4
+    ion_number, n_eff_upper, n_eff_lower, electron_density, temperature = (
+        int(ion_number),
+        float(n_eff_upper),
+        float(n_eff_lower),
+        float(electron_density),
+        float(temperature),
     )
-    c4 = c4_prefactor * (
-        (n_eff_upper * ((5 * n_eff_upper**2) + 1)) ** 2
-        - (n_eff_lower * ((5 * n_eff_lower**2) + 1)) ** 2
+    ion_number = ion_number * ion_number
```

Comment:
this is not a good way of defining variables, we want to make sure people can read this later and not get confused :D